### PR TITLE
Develop/questions

### PR DESCRIPTION
**Pull request by hamanovich**

### CODE REVIEW

**Comment by hamanovich** [comment]
[Redesign (UI/UX) question card. Questions page: to hide answers](https://trello.com/c/29U73FSc/109-redesign-ui-ux-question-card-questions-page-to-hide-answers)

**Comment by hamanovich** [comment]
[When removing a user: remove or detach questions](https://trello.com/c/yIZGbzc4/118-when-removing-a-user-remove-or-detach-questions)

File: frontend/src/components/QuestionsPage/QuestionsTop.tsx [hunk 1]
```diff
@@ -35,18 +35,18 @@ const QuestionsTop: FunctionComponent<QuestionsTopProps> = ({ questions }) => (
           <BadgeGroup>
             {question.level &&
               question.level.map((level: string) => (
-                <Badge variant="primary" key={level}>
-                  {level}
-                </Badge>
+                <Fragment key={level}>
```
Comment (SW999) [inline frontend/src/components/QuestionsPage/QuestionsTop.tsx]: Do we really need Fragment here and below?

File: frontend/src/components/QuestionsPage/QuestionsWrapper.tsx [hunk 2]
```diff
@@ -48,6 +50,8 @@ const WrappedComponent = <P extends object>(
         this.getQueryQuestions(searchQuery);
```
Comment (SW999) [inline frontend/src/components/QuestionsPage/QuestionsWrapper.tsx]: Would it be better to use switch/case instead of else/if?

Comment (hamanovich) [reply frontend/src/components/QuestionsPage/QuestionsWrapper.tsx]: If/else is much faster ;)

File: frontend/src/components/QuestionsPage/QuestionsTop.tsx (hunk 1 above)
Comment (hamanovich) [reply frontend/src/components/QuestionsPage/QuestionsTop.tsx]: Yes, because I've added space after Badge and it counts as a separate 'item'

**Pull request merged**